api.py

```diff
@@ -57,53 +60,59 @@ async def find_mentions(body: ExtractorRequestBody) -> ExtractorResponse:
 
 
 @APP.get("/v1/get_mdr_comments", response_model=BaseResponse)
-def update_comments_from_mdr() -> None:
+def update_comments_from_mdr(
+    query: dict[str, Any] = Depends(MDRUpdateRequest.query_template)
+) -> None:
     """Get comments from mdr source, store them in the bucket and db."""
-    # get new new comments from mdr
-    # TODO
-    #raw_comments = get_raw_comments_from_mdr()
-    raw_comments = [Comment.dummy()]
+    config = MDRUpdateRequest.from_query(query)
+    get_comments = MDRCommentGetter()
```

Comment:
Note that here you are only extracting the *first page of comments* since the default value of "page" is equals to one. There might be multiple pages with items. I solved this with the following while:
page = 1
response = self.coral_request(from_date, to_date, page)
data = response['items']
while len(response['items']) > 0:
            page += 1
            response = self.coral_request(from_date, to_date, page)
            data.extend(response['items'])